pyriemann/datasets/sampling.py

```diff
@@ -129,24 +122,31 @@ def _rejection_sampling_2D(n_samples, sigma, random_state=None):
         Dispersion of the Riemannian Gaussian distribution.
     random_state : int, RandomState instance or None, default=None
         Pass an int for reproducible output across multiple function calls.
+    return_rate : boolean
```

Comment:
I think that calling it `return_acceptance_rate` would be better, since one could think that it is the `rejection_rate` or any other bizarre rate.

Reply:
Indeed, I've changed it.